mmcv/runner/hooks/logger/text.py

```diff
@@ -14,6 +20,12 @@ def log(self, runner):
             log_str = 'Epoch({}) [{}][{}]\t'.format(runner.mode, runner.epoch,
                                                     runner.inner_iter + 1)
         if 'time' in runner.log_buffer.output:
+            self.time_sec_tot += (runner.log_buffer.output['time'] *
+                                  self.interval)
+            time_sec_avg = self.time_sec_tot / (runner.iter + 1)
+            eta_sec = time_sec_avg * (runner.max_iters - runner.iter - 1)
+            eta_str = str(datetime.timedelta(seconds=int(eta_sec)))
+            log_str += ('eta: {}, '.format(eta_str))
```

Comment:
A minor comment: the brackets here are unnecessary since this line is short. I will merge this PR after the fix.

Reply:
thanks. I didnt notice that.